site/content/docs/howto/configure-supervisor.md

```diff
@@ -6,10 +6,11 @@ cascade:
 menu:
   docs:
     name: Configure Supervisor
```

Comment:
Maybe this title should be something like "Configure Supervisor as an OIDC Issuer" or something to distinguish it from the "Configure Supervisor With <Upstream>" pages.

Reply:
Thanks! Fixed in 9621ad9d2cba4cdaf7d3064a9841033696b57665.